### ISSUE TITLE
Reading table with Chinese / Latin characters mixed

### ISSUE THREAD

**Issue by mberneaud** [tabulapdf/tabula#488]
I'm trying to read in a table which contains text which is two columns for each cell. Once in Mandarin and once in English. Reading the table with tabula results in really wonky rows / columns, with mostly empty cells. All Latin characters disappear completely, while it looks like the Chinese ones get converted to random UTF-8 characters (e.g. æç ãê ;éê äê åê åã}~ è¤LãçêS}~ë). Using the lattice option results in a better output, but all characters still come out as gibberish. I would try setting the character encoding to "GBK", but I need the English text as I do not speak Mandarin. 

The offending pdf is this one here: http://www.hanban.edu.cn/report/pdf/2011_final.pdf (table on p. 51)

I'm using the current version of tabula (downloaded today) on Ubuntu 15.10. System locale is en_gb. 

edit: it might be something wonky specifically with the pdf I linked. The other ones I tried afterwards seem to work great.

**Comment by jeremybmerrill** [comment]
Are you using the Beta release or the current production release? 

It worked great for me (with no weird encoding chores, in en_us) with the Beta release.

It's very likely that switching to the Beta would make it work correctly: we've made a dramatic update to the extraction engine -- it's now totally rewritten. 

Let me know, happy to help more.

**Comment by mberneaud** [comment]
I am using the current production release (1.0.1). I'll give the Beta release a try tonight and tell you how it works later.

**Comment by jeremybmerrill** [comment]
@mberneaud How'd it go?

**Comment by mberneaud** [comment]
Sorry for never replying, I just edited my post for some reason. I tried the beta release and it didn't change anything in terms of the recognizing the characters correctly. I think it might be something that is related specifically to that document, as tabula worked great on other pdfs which contained both Latin and Chinese characters. In any case, keep up the good work, tabula has been a great help for me.
